src/module.ts

```diff
@@ -72,7 +72,7 @@ export default defineNuxtModule<SanityModuleOptions>(nuxt => ({
     contentHelper: true,
     imageHelper: true,
     dataset: 'production',
-    apiVersion: '1',
+    apiVersion: `${new Date().toISOString().split('T')[0]}`,
```

Comment:
The reason there is an API version at all is to prevent breaking changes, and to give some security and stability for users. Opting users into the latest version of the sanity client automatically is potentially quite dangerous.